### ISSUE TITLE
Breakage: image on https://tweakers.net and https://gathering.tweakers.net

### ISSUE THREAD

**Issue by crisp-tweakers** [easylist/easylist#14780]
### List the website(s) you're having issues:
`https://tweakers.net` and `https://gathering.tweakers.net`

### What happens?
Certain images don't work, e.g.:

`https://tweakers.net`
![image](https://user-images.githubusercontent.com/32640525/214511494-2a1d9f78-e3ef-4ead-988c-5fa1f4cb2739.png)

`https://gathering.tweakers.net/forum/list_messages/1964650`
![image](https://user-images.githubusercontent.com/32640525/214511859-b0502ce8-91cd-4a5f-84b7-9e4cb2615481.png)

External images are being proxied (to **avoid** tracking) and have the form of: `https://tweakers.net/camo/e26bc8c10e19731a4c982d1f100c1333b5d671fe/?url=https%3A%2F%2Fi.imgur.com%2FDiABngL.png`

### List Subscriptions you're using:
EasyList
EasyPrivacy
EasyDutch

### Your settings

- OS/version: windows 10
- Browser/version: Firefox 109.0
- Adblock Extension/version: uBlock Origin 1.46.0

### Other details:

Supect is the following rule from pull request https://github.com/easylist/easylist/pull/14760 for EasyPrivacy:

`||tweakers.net/*/?$image`

**Comment by crisp-tweakers** [comment]
@krystian3w no, this has nothing to do with the GDPR dialog

**Comment by crisp-tweakers** [comment]
Duplicate: https://github.com/easylist/easylist/issues/14843

uBlock already disabled this rule: https://github.com/uBlockOrigin/uAssets/issues/16585 - https://github.com/uBlockOrigin/uAssets/commit/5ad93ed4dfef5903edb9779426be59c9c2dc1f5c

**Comment by crisp-tweakers** [comment]
I don't think https://github.com/easylist/easylist/commit/8a99d705ab0d931ab90c53e383b48c51b5310168 will solve this issue

**Comment by krystian3w** [comment]
Yes, the filter from unbreak no longer has a match to deactivate the newer version of image filtering.

**Comment by crisp-tweakers** [comment]
People start to complain on our forum: https://gathering.tweakers.net/forum/list_messages/2171958

**Comment by JohnyP36** [comment]
See my comment https://github.com/uBlockOrigin/uAssets/issues/16585#issuecomment-1412298668 and https://github.com/EasyDutch-uBO/EasyDutch/issues/35#issuecomment-1412292353

It's better to not block those images it at all (for now).

**Comment by krystian3w** [comment]
Maybe needed is implement random numbers without parameters:

https://camo.githubusercontent.com/83e4bc8230521d2f55917aa2a05119dce726360d014885f1a21a285c8304ad1a/68747470733a2f2f692e696d6775722e636f6d2f394a69716c50542e706e67

**Comment by crisp-tweakers** [comment]
@krystian3w but to what point? Why should we change our implementation? It would not change the function, but users won't be able to identify the souce of the images anymore. It is still totally unclear to me why these images are being blocked in the first place.

**Comment by krystian3w** [comment]
For geeks can be left `data-src` https://stackoverflow.com/questions/15320052/what-are-all-the-differences-between-src-and-data-src-attributes or implement title attribute on hover.

Perhaps EasyPrivacy does not condone your database having information about the time loaded photo and otherwise IP of a visitor to a thread.

**Comment by krystian3w** [comment]
I meant first party tracking may also bad for EasyPrivacy.

That's why I described that you can make a list of what clients IPs have downloaded a proxied image and it doesn't appeal to the list maker.

Rather, you're burned that your server doesn't record for itself what client IP downloaded the proxied image.

**Comment by crisp-tweakers** [comment]
@krystian3w whynwould we do that? We have no need for such information, especially not on a user-based level. We are a European-based company and as such are bound by privacy regulations such as the GDPR. This requires us to inform users about all processing of user-based data, and even ask consent when we have no direct legitimate interest in processing such data.

We are very strict in following those regulations. We think privacy and security are important for our users. That is also why our advertising is absolutely free from 3rd-party tracking, and by default we won't include any 3rd-party embedded items such as YouTube video's unless the user consents to that.

We understand why users may want to block banners, even when they are free of tracking. We don't like it, because they make up a substantial part of our income, and we will try to prevent it, but when it comes to privacy there should be nothing to block on our site. If someone believes there is we would love to hear it and discuss it, instead of having features from our site blocked and users complaining about it.

**Comment by crisp-tweakers** [comment]
I see the rule has been remove: https://github.com/easylist/easylist/commit/a8cded6675c28d7a4ef8b819396c40d1a6a8ae70

Thank you. We shall tell users to update their lists when they're not willing to whitelist our site.
